scripts/pipeline/fyre-e2e.sh

```diff
@@ -155,6 +125,47 @@ main() {
     return $result
 }
 
+install_operator() {
+    # Apply the catalog
+    echo "****** Applying the catalog source..."
+    cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: websphere-liberty-catalog
+  namespace: $TEST_NAMESPACE
+spec:
+  sourceType: grpc
+  image: $CATALOG_IMAGE
+  displayName: WebSphere Liberty Catalog
+  publisher: IBM
+EOF
+
+    echo "****** Applying the operator group..."
+    cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: websphere-operator-group
+  namespace: $TEST_NAMESPACE
+EOF
+
+    echo "****** Applying the subscription..."
+    cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: websphere-liberty-operator-subscription
+  namespace: $TEST_NAMESPACE
+spec:
+  channel:  v1.0
```

Comment:
It would be great if the default channel (defined [here](https://github.com/WASdev/websphere-liberty-operator/blob/cc5460738fffed01a03e138922221034a6ee3851/Makefile#L32) in the Makefile) is automatically used here, so we don't forget/have to manually update it.